### PR TITLE
dbeaver/pro#2448 Usage statistics: check the box to not send the statistics

### PR DESCRIPTION
**Pull request by serjiokov**
Usage statistics: check the box fixed

### CODE REVIEW

**Review by E1izabeth — APPROVED** [review]

**Comment by serjiokov** [comment]
image

File: plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/StatisticsCollectionConfirmDialog.java [hunk 1]
```diff
@@ -59,20 +63,23 @@ protected Composite createDialogArea(@NotNull Composite parent) {
                         null,
                         PrefPageUsageStatistics.PAGE_ID);
                 }));
-
+        UIUtils.createEmptyLabel(composite, 1,1);
+        shareData = UIUtils.createCheckbox(composite, "Ask not to share", false);
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/StatisticsCollectionConfirmDialog.java]: This sentence sounds a bit weird. Was it discussed prior to this change?

Something like "I do not want to share anonymous statistics" or "Don't share" would fit better, but it's up for discussion.

Comment (E1izabeth) [reply plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/StatisticsCollectionConfirmDialog.java]: Agreed

Comment (serjiokov) [reply plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/StatisticsCollectionConfirmDialog.java]: No objections from my side, let's discuss with Anna.

Comment (arhayka) [reply plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/StatisticsCollectionConfirmDialog.java]: Ok, I agree. Please change it to "I don't want to share the data".
I hope it looks better.

Comment (serjiokov) [reply plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/StatisticsCollectionConfirmDialog.java]: Fixed

Comment (ShadelessFox) [reply plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/StatisticsCollectionConfirmDialog.java]: Please update the screenshot as well

Comment (serjiokov) [reply plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/StatisticsCollectionConfirmDialog.java]: image

**Review by E1izabeth — APPROVED** [review]

File: plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/PrefPageUsageStatistics.java [hunk 2]
```diff
@@ -53,7 +54,7 @@ public void init(IWorkbench workbench) {
     protected Control createPreferenceContent(@NotNull Composite parent) {
         Composite composite = UIUtils.createPlaceholder(parent, 1);
 
-        Composite group = UIUtils.createControlGroup(composite, "Data sharing", 1, GridData.FILL_HORIZONTAL, SWT.DEFAULT);
+        Composite group = UIUtils.createControlGroup(composite, "Data share", 1, GridData.FILL_HORIZONTAL, SWT.DEFAULT);
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/PrefPageUsageStatistics.java]: Why this change?

Comment (serjiokov) [reply plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/PrefPageUsageStatistics.java]: by Anna requiest

Comment (ShadelessFox) [reply plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/PrefPageUsageStatistics.java]: In my opinion, it's not a suitable term. The term "data sharing" refers to the concept as a whole. On the other hand, the term "data share" refers to a particular sample of shared data.

File: plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/StatisticsCollectionConfirmDialog.java (hunk 1 above)
Comment (arhayka) [reply plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/StatisticsCollectionConfirmDialog.java]: sorry guys, we have to do it again =) please change the checkbox label to **Do not share data**

File: plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/PrefPageUsageStatistics.java (hunk 2 above)
Comment (ShadelessFox) [reply plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/PrefPageUsageStatistics.java]: We discussed that this change is redundant. Please revert it.

Comment (arhayka) [reply plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/PrefPageUsageStatistics.java]: We don't have to change this heading in Preferences. I'm not sure that this is so important, but here Data sharing looks good.

File: plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/StatisticsCollectionConfirmDialog.java (hunk 1 above)
Comment (serjiokov) [reply plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/StatisticsCollectionConfirmDialog.java]: image

Comment (serjiokov) [reply plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/StatisticsCollectionConfirmDialog.java]: image

**Comment by LonwoLonwo** [comment]
But what about localization? It looks like a good place to add it.

**Comment by serjiokov** [comment]
image

**Comment by serjiokov** [comment]
image

**Comment by ShadelessFox** [comment]
Localization is hardcoded. @LonwoLonwo suggests adding proper localization keys for these strings, at least for English, to allow future contributors to translate these as well.

**Comment by DariaMarkaryan** [comment]
Verified
